### DISCUSSION TITLE
Transition from V1 to V2 case?

### DISCUSSION THREAD

**Discussion by d-nx** [doudar/SmartSpin2k#403] [General]
Has anybody successfully ported a V1 (non PCB) to V2 case? Looks like the V2 case is PCB only, I'm curious if someone's done it without the PCB, or if there's a bracket I can print to make a non PCB fit into the V2?

My V1 case works great, and I have all the pieces I need to make it work, so I'm hoping to be able to port it all to the new case.

**Comment by doudar** [comment]
I would think you could but it might be tight. Don't use an ultra short 😅. I'd also recommend the direct mount case unless your head tube is very close.

**Reply by d-nx** [reply]
It would be for an echelon sport.

**Comment by eMadman** [comment]
Definitely use the direct mount mod then.  We have a mount for the echelon
connect sport.

**Comment by d-nx** [comment]
Thanks guys. Seems like it might be better to do a PCB build for the new case. Any chance someone out here has 1 board to sell? PCBway has a 5 minimum and I figure I'd ask =)

**Reply by eMadman** [reply]
Email me and I can sort you out.  emadmanmakes@gmail.com

**Reply by d-nx** [reply]
Thanks so much! I sent you an email with my info.

**Comment by d-nx** [comment]
This thing came out great!

Thanks so much for your help sourcing the electronics @eMadman ! The soldering jig helped me A LOT!

![20221130_122729](https://user-images.githubusercontent.com/80003983/204876644-503cba1a-12da-4d89-9e91-88caccfd321f.jpg)
![20221130_122734](https://user-images.githubusercontent.com/80003983/204876648-f2be943f-f1e5-43e7-9e07-aa902d9bec4c.jpg)
![20221130_122741](https://user-images.githubusercontent.com/80003983/204876652-a4c31cf9-ffdf-4f7f-8b5d-a7561904a796.jpg)
![20221130_122753](https://user-images.githubusercontent.com/80003983/204876657-98d0aabd-e0cc-4c15-893e-f12402869fb4.jpg)
![20221130_122814](https://user-images.githubusercontent.com/80003983/204876660-8e3ccfbf-f127-4db7-8fdc-16e2d553b6aa.jpg)
![20221130_122819](https://user-images.githubusercontent.com/80003983/204876665-52b50034-a4d0-45d6-8f3e-f12bafe0d10d.jpg)
![20221130_122829](https://user-images.githubusercontent.com/80003983/204876667-6d6107be-0240-41b7-a9e9-58d4c0135aa4.jpg)
![20221130_122849](https://user-images.githubusercontent.com/80003983/204876673-b39da78b-31dd-4ca2-8d91-f329524f9b0a.jpg)
![20221130_124449](https://user-images.githubusercontent.com/80003983/204876676-d28be7f0-ddbf-4116-b75c-5cf0333b942d.jpg)
![20221130_130742](https://user-images.githubusercontent.com/80003983/204876680-c74d95d3-b8e9-4d52-9357-21a9a3101374.jpg)
![20221130_130746](https://user-images.githubusercontent.com/80003983/204876683-eb5938c1-c943-429d-8b65-5fb1278246e4.jpg)

**Reply by eMadman** [reply]
Looks fantastic!  And I'm glad you found the solder jig to be valuable.   

If it's working fine, ignore my nit picking, carry on and enjoy the rides.  One small suggestion:  Swap the thicker size nut for the jam nut in the recess at the bottom of the knob cup.  The spacing there is such that you shouldn't need the washer.  A bit of threadlocker or nail polish will keep the jam nut from coming loose during regular use.

For my own reference - how much slack is remaining in the arm?  Should I make the arm longer so there's more adjustment room?  It looks like it might be just a bit shorter than ideal for the knob to head tube distance on this bike.

**Reply by doudar** [reply]
Sweet pictures!!!!!!!

Thanks for posting them! I love to see other people's work.

Thank you!!

**Reply by d-nx** [reply]
The arm is maxed out. I think an extra 10mm would be great.

I'll swap the nuts around, that makes a lot of sense. I don't like the washer in there either ;)

I'm waiting for some grey TPU to make new shifters. You know what would be great, some plugs to cover the screw holes. I would print those in TPU also so they're easy to remove if you need to open up the case.

**Reply by d-nx** [reply]
Fixed the screw and snapped a new pic that includes the motor.

Cheers!

![20221130_142813](https://user-images.githubusercontent.com/80003983/204892147-fcd1b5dc-75cd-4cc2-9629-0e9b0264e6b3.jpg)

**Reply by doudar** [reply]
Wow! It looks absolutely beautiful. 

Would you mind if we used the photo in the wiki version 2.0 building section?

**Reply by doudar** [reply]
If you could as well, I'd love it if you tried the firmware from the auto reconnect PR on your Echelon. If you save your echelon on the BLE scanner page and then either save your HR monitor or select none (it currently doesn't work with "any"), it should always connect to your Echelon when it sees it (It checks every 30 seconds).  

It's this message: https://github.com/doudar/SmartSpin2k/pull/405#issuecomment-1328416065

**Reply by d-nx** [reply]
> Wow! It looks absolutely beautiful.
> 
> Would you mind if we used the photo in the wiki version 2.0 building section?

Go for it! I don't mind at all.

> if you could as well, I'd love it if you tried the firmware from the auto reconnect PR on your Echelon. If you save your echelon on the BLE scanner page and then either save your HR monitor or select none (it currently doesn't work with "any"), it should always connect to your Echelon when it sees it (It checks every 30 seconds).

I'll give it a shot.

**Reply by doudar** [reply]
> The arm is maxed out. I think an extra 10mm would be great.
> 
> I'll swap the nuts around, that makes a lot of sense. I don't like the washer in there either ;)
> 
> I'm waiting for some grey TPU to make new shifters. You know what would be great, some plugs to cover the screw holes. I would print those in TPU also so they're easy to remove if you need to open up the case.

Or you all could just use the black oxide countersunk wood screws, but I do have to admit the brass inserts look nice when it's open :)
